enos/modules/vault_cluster/scripts/enable_audit_logging.sh

```diff
@@ -4,7 +4,10 @@
 set -eux
 
 # Run nc to listen to port 9090 for the socket audit log
-nohup nc -l 9090 &>/dev/null </dev/null &
+nc -l 9090 &>/dev/null &
```

Comment:
I would prefer we continue to not expect a hangup when backgrounding the process in an SSH session. I also wrote to STDIN to ensure that nc always gets an EOF immediately and thus would no expect anything to be written to STDIN. The prior behavior with the sleep seems like a decent-ish solution. Though I'd prefer we instead verify that we can connect to the socket on 9090.

Reply:
We'll merge this to get the pipeline moving and I'll work on what I'm thinking in another PR